src/routes/trade/trade.module.scss

```diff
@@ -78,68 +108,93 @@
     }
   }
   &__buy {
-    background-color: #3d9494;
+    background-color: #4ab5b3;
```

Comment:
Can u move these colors to theme file and use the variables. Also please check if they need any change in dark and light mode

Reply:
updated.